### ISSUE TITLE
Broken tabs after reopening closed ones

### ISSUE THREAD

**Issue by octohedron** [brave/browser-laptop#9245]
Description: I had many tabs open, closed some of them, then reopened them with Shift + Cmd + T and now I have two tabs that show as "Untitled" and are unresponsive and I can't close them or switch to them, not even with Ctrl + tab, hovering over their titles shows a grey window (like an empty website)

- Did you search for similar issues before submitting this one?

Yep

**Expected behavior:**

These tabs loading or switching to the tab or being able to close the tabs, some tabs still work tho.

- Platform:

macOS Sierra 10.12.5

- Brave Version:

```
Brave                             0.15.314
rev                               75ffa36
Muon                              3.0.202
libchromiumcontent                58.0.3029.110
V8                                5.8.283.38
Node.js                           7.9.0
Update Channel                    dev
os.platform                       darwin
os.release                        16.6.0
os.arch                           x64
```
- Steps to reproduce:
    1. Open many tabs
    2. Close some
    3. Shift + Cmd + T

- Will the steps above reproduce in a fresh profile? If not what other info can be added?

Nope

- Is this an issue in the currently released version?

Probably

- Can this issue be consistently reproduced?

Not that I know of

- Screenshot if needed:

<img width="728" alt="screen shot 2017-06-03 at 22 48 21" src="https://cloud.githubusercontent.com/assets/7332540/26756972/e5f15642-48ae-11e7-9f33-1e2b6200095b.png">

**Comment by bsclifton** [comment]
Closing this issue for now

@octohedron if you have specific steps to reproduce (with a current version of Brave), please do re-open and share. With more specific steps, we should be able to track down the issue (if it still occurs).  Thanks! 😄

**Comment by octohedron** [comment]
Unfortunately, after choosing Brave as my main browser on my MacBook for quite a while I had to switch back to chrome because of too many tabs bugs, for example, many times I had to click on the active tab label to be able to switch tabs, a somewhat related problem to this issue. I might give it another try in a couple of months or years? If it looks more stable, tested and feature complete.

It is and will remain my main browser in Android though.
